addon/content/components/compose/composeWidget.css

```diff
@@ -64,18 +64,19 @@ input:focus {
 }
 
 .compose > .buttons {
-  margin-top: 10px;
+  padding-top: 10px;
+  display: flex;
+  justify-content: space-between;
 }
 
 .compose > .buttons > button {
-  padding: 0 5px;
-  margin-inline-end: 3px;
-  background-color: white;
+  padding: 0.3em 1em;
+  background-color: -moz-dialog;
+  color: -moz-dialogtext;
```

Comment:
This seems fine in dark mode (though maybe a little bright?), but in light mode it is too harsh compared to the rest of the UI.

Normally in light mode we're using `#666`: https://github.com/thunderbird-conversations/thunderbird-conversations/blob/1f5a213c734bafc535587a6d65cf54166865a4f4/addon/content/conversation.css#L152

So I think we want to have `#666` as the default and `-moz-dialogtext` as the `(prefers-color-scheme: dark)` option.

Reply:
Well if that's what you normally do, then of course use 666.

However, maybe all of your UI should eventually be consistent with the user theme. But that's work for other PR...

Reply:
Yeah, let's be consistent for now, and we can explore different options in later options.